src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -24,40 +62,17 @@ class Breadcrumb extends PureComponent {
   render() {
     const { createURL, items, refine, cssClasses } = this.props;
 
-    const breadcrumb = items.map((item, idx) => {
-      const isLast = idx === items.length - 1;
-      const label = isLast ? (
-        <a className={`${cssClasses.disabledLabel} ${cssClasses.label}`}>
-          {item.name}
-        </a>
-      ) : (
-        <a
-          className={cssClasses.label}
-          href={createURL(item.value)}
-          onClick={e => {
-            e.preventDefault();
-            refine(item.value);
-          }}
-        >
-          {item.name}
-        </a>
-      );
+    const breadcrumb = items.map(renderLink(this.props));
 
-      return [
-        <Template
-          key={item.name + idx}
-          rootProps={{ className: cssClasses.separator }}
-          templateKey="separator"
-          {...this.props.templateProps}
-        />,
-        label,
-      ];
-    });
+    const homeClassnames =
+      items.length === 0
+        ? cx(cssClasses.item, cssClasses.selectedItem)
+        : cx(cssClasses.item);
 
-    const homeClassNames =
-      items.length > 0
-        ? [cssClasses.home, cssClasses.label]
-        : [cssClasses.disabledLabel, cssClasses.home, cssClasses.label];
+    const rootClassnames =
+      items.length === 0
```

Comment:
You could use the same trick mentioned before to conditionally apply some classes.